multidict/_multidict.c

```diff
@@ -572,7 +572,7 @@ multidict_sq_contains(MultiDictObject *self, PyObject *key)
 static inline PyObject *
 multidict_tp_iter(MultiDictObject *self)
 {
-    return PyObject_GetIter(multidict_keysview_new((PyObject*)self));
```

Comment:
@Reskov could you please explain the implications of dropping `PyObject_GetIter()`?

Reply:
@webknjaz Suddenly, I'm not familiar with python c-api, as far as I get `PyObject_GetIter` returns new iterator from the object `multidict_keysview_new`, which is actually equivalent to call multidict_keys_iter_new.

So in other words, I just eliminate unnecessary creation of the keysview structure, but result of the call should be the same therefore shouldn't have any impact.

Also my code-snippet from the issue now shows, that fix helps
```
Multidict: 4.7.3
Unreleased objects:
```

P.S. anyway I'm waiting for the review, because I'm not sure that I was right.

Reply:
That's probably because you don't create an object. Usually the trick is to deallocate objects after the usage. If I were to guess I'd say look for a missing decref

Reply:
@asvetlov @gyermolenko should a Py_DECREF be added somewhere here https://github.com/aio-libs/multidict/blob/dda85cf/multidict/_multidict.c#L637 instead?